docs/requirements/stakeholder/index.rst

```diff
@@ -264,9 +265,91 @@ Dependability
    * ECC Memory
    * Software Lockstep
    * Power management integrated circuit (PMIC), external watchdog and voltage monitoring
-   * Safe switch from engineering for field mode and back
+   * Safe switch from engineering to field mode and back
 
-   Note: This is part of 0.5 release and therefore can only support ASIL_B. Goal is ASIL_D.
+
+.. stkh_req:: SW-platform error reaction
+   :id: stkh_req__dependability__error_reaction
+   :reqtype: Functional
+   :security: NO
+   :safety: ASIL_B
+   :rationale: Safety relies on error reporting, e.g. to be able to reach a safe system state.
+   :status: valid
+   :tags: safety_mechanism
+
+   The SW-platform shall react in the following way on errors:
+
+   - report the error (incl. details if available) to the calling user function
+
+   Note: See also :need:`stkh_req__dependability__safe_state`
+
+
+.. stkh_req:: Safe SW-platform state
+   :id: stkh_req__dependability__safe_state
+   :reqtype: Functional
+   :security: NO
+   :safety: ASIL_B
+   :rationale: A safe state definition is expected by every safety standard.
```

Comment:
```suggestion
   :rationale: A safe state definition is a common expectation of safety integrity standards.
```
I wonder which safe state definition the "safety standard for toys" or "flame resistant fabrics" will have. Also IEC 61508 or ISO 26262 are mainly safety integrity standards and to plain safety standards.